_includes/2020/KeymanDownloadVersions.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
This class may not last long in this form; `KeymanDownloadVersions` is used currently *only* for macOS downloads and that will probably disappear as well once we finish the universal link model for macOS.